pil_analyzer/src/pil_analyzer.rs

```diff
@@ -191,6 +191,38 @@ impl<T: FieldElement> PILContext<T> {
             PilStatement::ConstantDefinition(_, name, value) => {
                 self.handle_constant_definition(name, value)
             }
+            PilStatement::LetStatement(start, name, None) => {
+                // Handle all let statements without assignment as witness column declarations for now.
+                self.handle_polynomial_declarations(
```

Comment:
You could use `handle_polynomial_definition` directly, which is equivalent to calling `handle_polynomial_definition` with a vector of size 1.